network/dropper.go

```diff
@@ -54,6 +54,7 @@ func handleDropping(log zerolog.Logger, wg *sync.WaitGroup, cfg *Config, peers p
 		addresses := peers.Addresses()
 		address := addresses[rand.Int()%len(addresses)]
 		err := peers.Drop(address)
+
```

Comment:
This new line should not be there :D

Reply:
haha, I saw it yesterday, going to remove it 👍